src/config/index.ts

```diff
@@ -49,4 +63,8 @@ export default {
     dbName: process.env.DATABASE || 'readitla_ril-tmp',
     tz: process.env.DATABASE_TZ || 'US/Central',
   },
+  queueDelete: {
+    queryLimit: 500,
+    itemIdChunkSize: 100,
```

Comment:
20, per slack discussion https://pocket.slack.com/archives/C03E28D1GUD/p1657671668758909

Reply:
#266 updates this